nanome_rmsd/rmsd_menu.py

```diff
@@ -267,20 +314,46 @@ def target_tab_pressed_callback(button):
         # no hydrogen = ! no hydrogen
         def no_hydrogen_button_pressed_callback(button):
             no_hydrogen_button.selected = not no_hydrogen_button.selected
+            if no_hydrogen_button.selected:
+                no_hydrogen_text.text_color = Color.from_int(0x00ecc4ff)
```

Comment:
we should have a constant color at the top instead of continually making these with magic numbers.